### ISSUE TITLE
django-ordered-model 3.7.2 breaks migrations

### ISSUE THREAD

**Issue by tomasvlt** [maykinmedia/django-admin-index#82]
Attempting to `python manage.py migrate` or `makemigrations` on a fresh install returns the following error:

```
SystemCheckError: System check identified some issues:

ERRORS:
AppGroup: (ordered_model.E003) OrderedModelBase subclass has a ModelManager that does not inherit from OrderedModelManager.
AppLink: (ordered_model.E003) OrderedModelBase subclass has a ModelManager that does not inherit from OrderedModelManager.
```

A temporary workaround is to define a requirement of `django-ordered-model==3.7.1`.
I've not investigated further, but I imagine this can be properly solved by adding the class inherit.

**Comment by swehba** [comment]
I had a similar issue. In my case, Django reported the following error:

`OrderedModelBase subclass has a ModelManager that does not inherit from OrderedModelManager`

Our project does not load django-ordered-model directly. It is loaded via django-admin-index. The developer of django-ordered-model explained the issue as follows:

> this is likely to be an issue with the way your project uses this library. The check here is new in 3.7, but the misconfiguration it identifies may have existed for some time.
> 
> To resolve this and allow the new code to work there [are instructions in our README](https://github.com/django-ordered-model/django-ordered-model#custom-manager-and-queryset) explaining that the ModelManager needs to inherit from our base class. You can pinpoint the relevant section by the error code that will have been printed later in the Check failure output.

As a work-around we fixed the version of `django-ordered-model` to 3.6. Given the message from @tomasvlt, it looks like we could also have fixed the version to 3.7.1.

**Comment by shuckc** [comment]
I've dropped this to a Warning in `django-ordered-model` 3.7.4 which is up on PyPi.
